drivers/gpio/gpio_b91.c

```diff
@@ -461,8 +461,10 @@ static void gpio_b91_irq_handler(const struct device *dev)
 {
 	struct gpio_b91_data *data				= dev->data;
 	const struct gpio_b91_config *cfg		= dev->config;
+#ifdef CONFIG_PM_DEVICE
```

Comment:
Please replace it with CONFIG_PM instead of CONFIG_PM_DEVICE

Reply:
Done